packages/tenant-process/src/model/domain/errors.ts

```diff
@@ -24,6 +24,9 @@ export const errorCodes = {
   certifiedAttributeAlreadyAssigned: "0015",
   attributeVerificationNotAllowed: "0016",
   verifiedAttributeSelfVerification: "0017",
+  attributeRevocationNotAllowed: "0018",
+  attributeAlreadyRevoked: "0019",
+  verifiedAttributeSelfRevocation: "0020",
```

Comment:
Maybe `verifiedAttributeSelfRevocationNotAllowed`? Probably it's too verbose, but the meaning it's clear.
Otherwise `selfRevocationNotAllowed`?